cdap-watchdog/src/main/java/co/cask/cdap/logging/appender/LogAppenderInitializer.java

```diff
@@ -87,4 +98,15 @@ public void close() {
       LOG.info("Done stopping log appender {}", logAppender.getName());
     }
   }
+
+  private LoggerContext getLoggerContext() {
+    ILoggerFactory loggerFactory = LoggerFactory.getILoggerFactory();
+    // TODO: fix logging issue in mapreduce:  ENG-3279
```

Comment:
Not sure, cannot find the JIRA number 3279 related to logging issue in mapreduce